Solutions/Darktrace/Workbooks/DarktraceWorkbook.json

```diff
@@ -0,0 +1,2628 @@
+              "size": 0,
+              "title": "Top 10 Highest Severity SaaS Model Breaches",
+              "timeContextFromParameter": "Timeframe",
+              "queryType": 0,
+              "resourceType": "microsoft.operationalinsights/workspaces",
+              "visualization": "table",
+              "gridSettings": {
+                "formatters": [
+                  {
+                    "columnMatch": "TimeGenerated",
+                    "formatter": 0,
+                    "formatOptions": {
+                      "customColumnWidthSetting": "20%"
+                    }
+                  },
+                  {
+                    "columnMatch": "Activity",
+                    "formatter": 1,
+                    "formatOptions": {
+                      "linkColumn": "DarktraceURL",
+                      "linkTarget": "Url",
+                      "customColumnWidthSetting": "40%"
+                    }
+                  },
+                  {
+                    "columnMatch": "DeviceName",
+                    "formatter": 0,
+                    "formatOptions": {
+                      "customColumnWidthSetting": "17.5%"
+                    }
+                  },
+                  {
+                    "columnMatch": "DeviceAddress",
+                    "formatter": 0,
+                    "formatOptions": {
+                      "customColumnWidthSetting": "17.5%"
+                    }
+                  },
+                  {
+                    "columnMatch": "Severity",
+                    "formatter": 8,
+                    "formatOptions": {
+                      "min": 1,
+                      "max": 10,
+                      "palette": "yellowOrangeRed"
+                    }
+                  },
+                  {
+                    "columnMatch": "DarktraceURL",
+                    "formatter": 5
+                  }
+                ],
+                "labelSettings": [
+                  {
+                    "columnId": "TimeGenerated",
+                    "label": "Time"
+                  }
+                ]
+              },
+              "sortBy": []
+            },
+            "name": "Top 10 hitting SaaS devices"
+          },
+          {
+            "type": 3,
+            "content": {
+              "version": "KqlItem/1.0",
+              "query": "darktrace_model_alerts_CL\r\n| where dtProduct_s ==\"Policy Breach\" and (modelName_s contains (\"SaaS\") or modelName_s contains (\"IaaS\")) and compliance_s == true\r\n| make-series count() default=0 on TimeGenerated in range({Timeframe:start}, now(), {Timeframe:grain})",
```

Comment:
@JustasZDT,  compliance_s should be  compliance_b, please check and update, thanks.

Reply:
This has been updated now.